components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js

```diff
@@ -0,0 +1,327 @@
+
+                        $(column).each(function () {
+                            if ($(column.nodes()).attr('data-search')) {
+                                var titles = [];
+                                column.nodes().unique().sort().each(function (d, j) {
+                                    var title = $(d).attr('data-display');
+                                    if ($.inArray(title, titles) < 0) {
+                                        titles.push(title);
+                                        if (title !== undefined) {
+                                            select.append('<option value="' + title + '">' + title + '</option>')
+                                        }
+                                    }
+                                });
+                            } else {
+                                column.data().unique().sort().each(function (d, j) {
+                                    select.append('<option value="' + d + '">' + d + '</option>')
+                                });
+                            }
+                        });
+                    } else if (filterColumn.eq(column.index()).hasClass('text-filter')) {
+                        var title = filterColumn.eq(column.index()).attr('data-for');
+                        $(filterColumn.eq(column.index()).empty()).html('<input type="text" class="form-control" placeholder="Search for ' + title + '" />');
+
+                        filterColumn.eq(column.index()).find('input').on('keyup change', function () {
+                            column
+                                .search($(this).val())
+                                .draw();
+                        });
+                    }
+
+                });
+
+                //Search input default styles override
+                var search_input = elem.closest('.dataTables_wrapper').find('div[id$=_filter] input');
+                search_input.before('<i class="fw fw-search search-icon"></i>').removeClass('input-sm');
+
+                // Create sorting dropdown menu for list table advance operations
+                var dropdownmenu = $('<ul class="dropdown-menu arrow arrow-top-right dark sort-list add-margin-top-2x"><li class="dropdown-header">Sort by</li></ul>');
+                $('.sort-row th', elem).each(function () {
+                    if (!$(this).hasClass('no-sort')) {
+                        dropdownmenu.append('<li><a href="#' + $(this).html() + '" data-column="' + $(this).index() + '">' + $(this).html() + '</a></li>');
+                    }
+                });
+
+                //Append advance operations to list table toolbar
+                $('.dataTable.list-table').closest('.dataTables_wrapper').find('.dataTablesTop .dataTables_toolbar').html('' +
+                    '<ul class="nav nav-pills navbar-right remove-margin" role="tablist">' +
+                    '<li><button data-click-event="toggle-select" class="btn btn-default btn-primary">Select</li>' +
```

Comment:
Please check whether these exceed the 120 limit

Reply:
Fixed with #1690